tests/integration/integration_python_test.go

```diff
@@ -2192,6 +2192,52 @@ func TestPythonComponentProviderException(t *testing.T) {
 	})
 }
 
+// Test that resource references work
+func TestPythonComponentProviderResourceReference(t *testing.T) {
+	// Manually set pulumi home so it's used for `plugin install`
+	pulumiHome := t.TempDir()
+	t.Logf("PULUMI_HOME = %s", pulumiHome)
+	t.Setenv("PULUMI_HOME", pulumiHome)
+	integration.ProgramTest(t, &integration.ProgramTestOptions{
+		PulumiHomeDir: pulumiHome,
+		PrepareProject: func(info *engine.Projinfo) error {
+			cmd := exec.Command("pulumi", "plugin", "install", "resource", "command", "1.1.0-alpha.1745259160")
```

Comment:
We need SDKs generated with 3.163.0 to have the `pulumi_type` available on resources. This pre-release of the `command` plugin has that.

Reply:
do we ever clean these up? Should we just wait for a proper release to depend on?

Reply:
It gets installed in a temporary $PULUM_HOME/plugins, and removed when the test ends.

> Should we just wait for a proper release to depend on?

Yeah, probably nicer to wait for a proper release than having the test rely on a pre-release. I think the command provider is ready for a new release, I'll check with providers.